ai/STA/Tactic/position_for_pass.py

```diff
@@ -67,5 +70,30 @@ def _get_destination_pose(self):
         ball_x = self.game_state.get_ball_position().x
         ball_y = self.game_state.get_ball_position().y
 
+        self._find_best_player_position()
+
         destination_orientation = np.arctan2(ball_y - player_y, ball_x - player_x)
         return Pose(self.target.position, destination_orientation)
+
+    def _find_best_player_position(self):
+        if self.auto_position:
+            if self.player.id == 1: #role is 'top_defence':
+                A = Position(GameState().const["FIELD_OUR_GOAL_X_EXTERNAL"], GameState().const["FIELD_Y_TOP"] )
+                B = Position(0, GameState().const["FIELD_Y_TOP"] / 3)
+                self.target.position = best_position_in_region(self.player, A, B)
+            elif self.player.id == 2: #player.role is 'bottom_defence':
+                A = Position(GameState().const["FIELD_OUR_GOAL_X_EXTERNAL"], GameState().const["FIELD_Y_BOTTOM"])
+                B = Position(0, GameState().const["FIELD_Y_BOTTOM"] / 3)
+                self.target.position = best_position_in_region(self.player, A, B)
+            elif self.player.id == 3: #player.role is 'top_offence':
+                A = Position(GameState().const["FIELD_THEIR_GOAL_X_EXTERNAL"], GameState().const["FIELD_Y_TOP"])
+                B = Position(0, 0)
+                self.target.position = best_position_in_region(self.player, A, B)
+            elif self.player.id == 4: #player.role is 'bottom_offence':
+                A = Position(GameState().const["FIELD_THEIR_GOAL_X_EXTERNAL"], GameState().const["FIELD_Y_BOTTOM"])
+                B = Position(0, 0)
+                self.target.position = best_position_in_region(self.player, A, B)
+            elif self.player.id == 5: #player.role is 'center':
+                A = Position(GameState().const["FIELD_OUR_GOAL_X_EXTERNAL"], GameState().const["FIELD_Y_BOTTOM"] / 3)
+                B = Position(0, GameState().const["FIELD_Y_TOP"] / 3)
+                self.target.position = best_position_in_region(self.player, A, B)
```

Comment:
Use a list to iterate throught those conditions, instead of that repetitive code